releasenotes/notes/fleet-setup-keep-yaml-comments-40d804ecd60e916c.yaml

```
@@ -0,0 +1,6 @@
+---
+enhancements:
+  - |
+    The Datadog Installer introduced in 7.65.0 now keeps YAML comments
+    when updating the Agent configuration during installation.
+    Some formatting changes can be expected.
```

To address both comments, how about:
```suggestion
    The Datadog Installer now keeps YAML comments
    when updating the Agent configuration during installation.
    While comment text is preserved, its format may not be.
    Changes to whitespace, indentation, or blank lines may occur.
```
Is it important to mention that the Datadog Installer was introduced in 7.65? If it isn't, I think we can go ahead and omit that so we can focus on this change.